language/stdlib/modules/libra_system.move

```diff
@@ -126,8 +113,8 @@ module LibraSystem {
     }
 
     // Return the size of the current validator set
-    public fun validator_set_size(): u64 acquires ValidatorSet {
-        Vector::length(&borrow_global<ValidatorSet>(0x1D8).validators)
+    public fun validator_set_size(): u64 {
+        Vector::length(&get_validator_set().validators)
```

Comment:
since this does a copy, is there anyway to actually get the reference instead? even if it is a bit more code?

Reply:
There isn't unfortunately. The problem is that move bytecode verifier currently has no way of returning a reference to a resource on chain due to the move's borrow checker rules. If we really want to avoid the copy here we need to merge the two modules so that we can do a in place modification here directly.